crypto/kyber/kem_kyber.h

```diff
@@ -25,65 +27,9 @@
-                     uint8_t *secret_key /* OUT */);
-
-int kyber512r3_encapsulate_deterministic(uint8_t *ciphertext       /* OUT */,
-                                         uint8_t *shared_secret    /* OUT */,
-                                         const uint8_t *public_key /* IN  */,
-                                         const uint8_t *seed       /* IN */);
-
-int kyber512r3_encapsulate(uint8_t *ciphertext       /* OUT */,
-                         uint8_t *shared_secret    /* OUT */,
-                         const uint8_t *public_key /* IN  */);
-
-int kyber512r3_decapsulate(uint8_t *shared_secret    /* OUT */,
-                         const uint8_t *ciphertext /* IN  */,
-                         const uint8_t *secret_key /* IN  */);
-
-int kyber768r3_keypair_deterministic(uint8_t *public_key  /* OUT */,
-                                     uint8_t *secret_key  /* OUT */,
-                                     const uint8_t *seed  /* IN */);
-
-int kyber768r3_keypair(uint8_t *public_key /* OUT */,
-                     uint8_t *secret_key /* OUT */);
-
-int kyber768r3_encapsulate_deterministic(uint8_t *ciphertext       /* OUT */,
-                                         uint8_t *shared_secret    /* OUT */,
-                                         const uint8_t *public_key /* IN  */,
-                                         const uint8_t *seed       /* IN */);
-
-int kyber768r3_encapsulate(uint8_t *ciphertext       /* OUT */,
-                         uint8_t *shared_secret    /* OUT */,
-                         const uint8_t *public_key /* IN  */);
-
-int kyber768r3_decapsulate(uint8_t *shared_secret    /* OUT */,
-                         const uint8_t *ciphertext /* IN  */,
-                         const uint8_t *secret_key /* IN  */);
-
-int kyber1024r3_keypair_deterministic(uint8_t *public_key  /* OUT */,
-                                      uint8_t *secret_key  /* OUT */,
-                                      const uint8_t *seed  /* IN */);
-
-int kyber1024r3_keypair(uint8_t *public_key /* OUT */,
-                      uint8_t *secret_key /* OUT */);
-
-int kyber1024r3_encapsulate_deterministic(uint8_t *ciphertext       /* OUT */,
-                                          uint8_t *shared_secret    /* OUT */,
-                                          const uint8_t *public_key /* IN  */,
-                                          const uint8_t *seed       /* IN */);
-
-int kyber1024r3_encapsulate(uint8_t *ciphertext       /* OUT */,
-                          uint8_t *shared_secret    /* OUT */,
-                          const uint8_t *public_key /* IN  */);
-
-int kyber1024r3_decapsulate(uint8_t *shared_secret    /* OUT */,
-                          const uint8_t *ciphertext /* IN  */,
-                          const uint8_t *secret_key /* IN  */);
+const KEM * get_legacy_kem_kyber512_r3(void);
+const KEM * get_legacy_kem_kyber768_r3(void);
+const KEM * get_legacy_kem_kyber1024_r3(void);
```

Comment:
Is there a reason for the weird spacing?
```suggestion
const KEM *get_legacy_kem_kyber512_r3(void);
const KEM *get_legacy_kem_kyber768_r3(void);
const KEM *get_legacy_kem_kyber1024_r3(void);
```